appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Share your tables with users and groups within your cloud.
 Have a good time and manage whatever you want.
 
 ]]></description>
-    <version>0.5.1</version>
+    <version>0.6.3</version>
```

Comment:
Probably stick with just 0.6.0-dev0 for now, during development you can just increment to dev1 and so on :)